FabricTestExample/babel.config.js

```diff
@@ -1,3 +1,7 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
+  plugins: [
```

Comment:
Why are those plugins needed?

Reply:
I loosely followed [these guidlines published by RN](https://reactnative.dev/docs/next/new-architecture-app-renderer-ios). I did not test if the project builds w/o them.

Reply:
Good catch, looks like they weren't necessary